### ISSUE TITLE
Migration to dotnet standard

### ISSUE THREAD

**Issue by baywet** [kenakamu/UCWA2.0-CS#29]
Since .net standard is the way forward for libraries, it'd be a good thing to migrate the project.
Advantages would be:
- Support of non Microsoft platforms
- General performances improvements
- Standardized set of API's available

https://docs.microsoft.com/en-us/dotnet/standard/net-standard#comparison-to-portable-class-libraries

**Comment by kenakamu** [comment]
I totally agree on this. Which .net standard version you are thinking about?

**Comment by baywet** [comment]
At this moment I am not sure, my understanding is the lowest version you target the more platforms might be compatible but the less apis you will have access to add part of the standard (and less libraries you might find). There's also the question of the available projects templates that evolved over time as well as the tooling associated to those.
So I think the goal is to find a sweet spot to avoid having too much trouble and/or rewriting too much code.
I believe version 1.1 added quite a few apis top the barbone v1 and brought a lot of improvements. So I guess I'd aim for that and see what happens if I had to do it.
